mlflow/utils/autologging_utils.py

```diff
@@ -7,6 +7,11 @@
 
 
 INPUT_EXAMPLE_SAMPLE_ROWS = 5
+FAILED_INPUT_EXAMPLE_PREFIX_TEXT = "Failed to gather input example: "
```

Comment:
Afaik, this is only defined as a module-level variable for testing purposes later on, where it's used to validate that warning messages appear as expected. Directly checking against this value in the test case leaves us to uncaught regressions: e.g., "FAILED_INPUT_EXAMPLE_PREFIX_TEXT" becomes `Failed to gather signature` due to some unintended line swapping, and we don't catch it because we're just checking that the content of `FAILED_INPUT_EXAMPLE_PREFIX_TEXT` appears in the text.

Accordingly, can we drop these constants and test against the warning messages using inlined strings?